pkg/trace/agent/obfuscate.go

```diff
@@ -90,6 +93,27 @@ func (a *Agent) obfuscateSpan(span *pb.Span) {
 			return
 		}
 		span.Meta[tagElasticBody] = o.ObfuscateElasticSearchString(span.Meta[tagElasticBody])
+	case "system":
```

Comment:
A dedicated config parameter must be added to enable/disable the new obfuscator. You can see how it's done for the other ones (e.g `a.conf.Obfuscation.Mongo.Enabled`).

Reply:
Added in 60faac7